### PR TITLE
Filters report based on minStudents. startdate default set to two weeks prior of current date

### PR DESCRIPTION
**Pull request by roperasher**
Changes:
-takes in minStudents parameter from url and uses it to filter out classes that have an attendance less than minStudents.
-sets startdate to two weeks prior of the current date if the url parameter is null
-sets attendance value to +9999 when encountering ENOTFOUND error.

reports.json might be done incorrectly. If so, I'll update it before merging.

### CODE REVIEW

File: server/cli.js [hunk 1]
```diff
@@ -1,115 +1,120 @@
...
-    getCommissionReport(options).then((report) => {
-      writeToFile(report, options);
-    });
-  } else if (argv.report === "attendance") {
-    getAttendanceReport(options.format, options.startDate, options.endDate).then((report) => {
-      writeToFile(report, options);
-    });
-  }
-}
+import {getCommissionReport} from "./reports/commission/commission-endpoint.js";
```
Comment (DanielBrewerPsu) [inline server/cli.js:1]: Man, these line ending changes are so lame. They obscure the real changes in this large mass of false positives. Oh, well. Hopefully we'll figure out how to get all the line changes consistent soon. I'll propose something at tonight's meeting.

File: server/reports/attendance/attendance-endpoint.js [hunk 2]
```diff
@@ -1,155 +1,167 @@
...
+  if (Number.isNaN(minStudents)) {
+    minStudents = MINSTUDENTS_DEFAULT;
+  }
+
+  getAttendanceReport(format, startdate, enddate, minStudents)
+    .then((report) => {
+      if (format == "csv") {
+        let fileName = "AttendanceReport.csv";
+        //todo: we should choose a filename that includes helpful info like the date range
+        response.setHeader("Content-Disposition", "attachment ; filename=\"" + fileName + "\"");
```
Comment (DanielBrewerPsu) [inline server/reports/attendance/attendance-endpoint.js:155]: Be careful to not overwrite Jason's filename changes here

Comment (roperasher) [reply server/reports/attendance/attendance-endpoint.js:155]: Thanks for catching that. I'm changing it right now.

**Comment by roperasher** [comment]
I'm going to close this pull request and create a new one. I stupidly force pushed and it caused changes I didn't realize would happen. The new PR will contain all changes prior to the forced push.

**Pull request closed without merging**